### DISCUSSION TITLE
Can not install JCB

### DISCUSSION THREAD

**Discussion by cyupid75** [joomengine/Joomla-Component-Builder#851] [Q&A] [answered]
Hi
I just want to install JCB to my local Joomla for development

Windows Vista, PHP 5

There is no error messages, only header for error appear without any messages,  and JCB does not show up at components menu.

In database already created componentvuilder table...

Any one who can help?

Thank you

**Comment by cpaschen** [comment]
How do you have Joomla running on your system?
WAMP, DockerImage, etc.?

Or are you trying to develop 'remotely' (i.e. Joomla on an external server)?

**Reply by cyupid75** [reply]
My system running on
Window vista
Xampp
Php 5
Joomla 3.10.4

It's only local system

I try to using other extensions and it's works...

**Comment by cpaschen** [comment]
Hmmm ... not completely sure. I've never had a problem installing it.
However, I'm wondering if it might be related to your version of PHP. 
The last PHP5 version stop getting support 3 years ago. I didn't think that 3.10 would run on PHP 5.
I'd first suggesting updating your PHP to at least PHP7.
Also, make sure you have a current version of MySQL.
Just because other extensions work on your configuration, doesn't mean that your site shouldn't be updated.
Be sure to review your system logs to see if anything there might explain what is happening.

Maybe Llewellyn could comment here about what the minimum PHP version is (I couldn't find it in any of the wiki or main page info).

**Reply by cyupid75** [reply]
Maybe from the php version... But php7 can not be install on windows vista...
I'll try to another pc/laptop with newest system

**Comment by cyupid75** [comment]
System Info

Windows vista 32bit
Xampp 3.2.2
Php 5.6.36
MySql 5.5
Joomla 3.10.4

Can I use this system to install JCB ?

after installation
- nothing happen
- no error message
![IMG_20220102_105046](https://user-images.githubusercontent.com/96953416/147865666-6fa1d808-a0e2-4354-95fd-48340f23eb76.jpg)
No message at  system messages container (only x button)

**Comment by cpaschen** [comment]
Well,I checked and it appears that you have the minimums required to run Joomla 3.10.x.
But if you are getting that red box, then you are getting an error.

Have you checked your log files?
Also, make sure that you have correct permissions set for your various Joomla directories.

I personally use Akeeba Admin Tools to check that sort of thing; however, no current version of Akeeba Admin Tools will run on your configuration that old version of PHP.

So, I would guess that it is related to either:
1. File access permissions
2. JCB doesn't support your version of PHP

Maybe Llewellyn can let us know the minimum php version for JCB.

And if you find something in your logs that seems to be a potential problem but you can't decipher it, let us know here and one of us can try to help you figure it out more.

BTW ... I'm guessing you probably know this, but JUST in case you don't - you're trying to write code for the web using an OS that is no longer supported, with a PHP version that is no longer supported and neither are getting security updates. This is not a good environment to write code. And also a bit of a problem getting help as few people use this combination and haven't for many years. If you can't afford to upgrade to the latest version of Windows, then you can migrate over to Linux for development for free (and it will probably run on whatever computer hardware you are using - possibly better than Windows).

**Comment by cyupid75** [comment, marked as answer]
Finally... 
The problem is from Xampp...
I must uninstall the xampp and then install again with "Run as administrator"..
Maybe this effect to permissions file..

And everything is normal.
I can install JCB too

Thank you for your effort Chris... really appreciate

**Comment by cpaschen** [comment]
AHHH ... I didn't know that you could even install it without admin permissions.
Glad it was an easy fix.
You'll love using JCB.

And also, you'll get a LOT out of the videos that Llewellyn has posted on youtube. Some of them are a bit old (and the JCB interface has changed) but the concepts are all good. 
And if you watch a video and can't figure something out (because the looks have changed) post here and we'll try to help.
